<ui version="4.0" >
 <class>FOVWidgetUi</class>
 <widget class="QWidget" name="FOVWidgetUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>172</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>20</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="mFieldOfViewGroup" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>131</height>
      </size>
     </property>
     <property name="title" >
      <string>Field of View</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QRadioButton" name="mFOVAspectVerticalRadio" >
        <property name="text" >
         <string/>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="mFOVAspectRatioLabel" >
        <property name="text" >
         <string>Aspect Ratio:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLineEdit" name="mFOVAspectRatioText" >
        <property name="minimumSize" >
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QLabel" name="mFOVVerticalwAspectLabel" >
        <property name="text" >
         <string>Vertical:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="mFOVHorizontalVerticalRadio" >
        <property name="text" >
         <string/>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="mFOVHorizontalLabel" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Horizontal:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLineEdit" name="mFOVHorizontalText" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="minimumSize" >
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="readOnly" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4" >
       <widget class="QLabel" name="mFOVVerticalwHorizontalLabel" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Vertical:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5" >
       <widget class="QLineEdit" name="mFOVVerticalwHorizontalText" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="minimumSize" >
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="5" >
       <widget class="QLineEdit" name="mFOVVerticalwAspectText" >
        <property name="minimumSize" >
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QPushButton" name="mFOVResetButton" >
        <property name="text" >
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mFOVAspectVerticalRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFOVAspectRatioLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>37</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel" >
     <x>101</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mFOVAspectVerticalRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFOVAspectRatioText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>37</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel" >
     <x>201</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mFOVAspectVerticalRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFOVVerticalwAspectLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>37</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel" >
     <x>283</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mFOVAspectVerticalRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFOVVerticalwAspectText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>37</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel" >
     <x>367</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mFOVHorizontalVerticalRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFOVHorizontalLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>37</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel" >
     <x>101</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mFOVHorizontalVerticalRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFOVHorizontalText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>37</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel" >
     <x>201</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mFOVHorizontalVerticalRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFOVVerticalwHorizontalLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>37</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel" >
     <x>283</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mFOVHorizontalVerticalRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFOVVerticalwHorizontalText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>37</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel" >
     <x>367</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
